celerdata/assets/service_checks.json

```
@@ -0,0 +1 @@
+[]
```

I believe you'll want to define the service checks in this file: Service check `celerdata_fe.prometheus.health` is submitted for FE component and `celerdata_be.prometheus.health` is submitted for BE component.

---

‌‌‌‌‌Thank you very much for your review work. I don't quite understand the purpose of the `service_checks.json` file, so I kept it consistent with `neo4j/assets/service_checks.json`.

---

‌In general, FE and BE are deployed on different machines. The openmetrics configuration of CelerData Integration needs to be set according to specific circumstances.

---

I'm adding this template as an example, please customize it to fill in the blanks/gaps:

```suggestion
[
    {
        "agent_version": "TBD",
        "integration": "CelerData",
        "check": "celerdata_fe.prometheus.health",
        "statuses": [
            "ok",
            "critical"
        ],
        "groups": [
            "host"
        ],
        "name": "Can Connect",
        "description": "Returns CRITICAL if the Agent cannot connect to CelerData to collect metrics. Returns OK otherwise."
    },
    {
        "agent_version": "TBD",
        "integration": "CelerData",
        "check": "celerdata_be.prometheus.health",
        "statuses": [
            "ok",
            "critical"
        ],
        "groups": [
            "host"
        ],
        "name": "Can Connect",
        "description": "Returns CRITICAL if the Agent cannot connect to CelerData to collect metrics. Returns OK otherwise."
    }
]

```

---

done